sdk/Cargo.toml

```
@@ -1,6 +1,6 @@
 [package]
 name = "c2pa"
-version = "0.18.1"
```

Please revert this change. The version number bump will happen automatically when we publish the next release. Leaving this here would cause a gap in versioning.

---

The message I go from the error said to make it 0.19.0.  Should I change that back?

---

Yes, please change it back and ignore the error. More (frustrating) info in Jira under CAI-3479.